README.md

```diff
@@ -420,6 +420,23 @@ Stops all Workers and rejects all `Promise`s for pending tasks.
 
 This returns a `Promise` that is fulfilled once all threads have stopped.
 
+### Method: `close([options])`
+
+* `options`:
+  * `force`: A `boolean` value that indicates whether to abort all tasks that 
+  are enqueued but not started yet. The default is `false`.
+  * `timeout`: A `number` time (in milliseconds) to wait for the pool to 
```

Comment:
I believe that it has better ergonomics if set when instantiating `Piscina`, so it looks like `new Piscina({ closeTimeout: 40000 })` wdyt?  🤔